frontend/src/components/Header/Header.tsx

```
@@ -3,31 +3,47 @@ import { ModeToggle } from "../mode-toggle";
-        <Link to="/" className="md:text-xl text-lg font-semibold font-serif">Project Sphere</Link>
+        <Link to="/" className="md:text-xl text-lg font-semibold font-serif">
+          Project Sphere
+        </Link>
 
         {/* Center: Navigation Links (Hidden in Mobile) */}
         <nav className="hidden md:flex space-x-6">
-          <a href="#" className="hover:text-gray-300">Home</a>
-          <a href="#" className="hover:text-gray-300">About</a>
-          <a href="#" className="hover:text-gray-300">Services</a>
-          <a href="#" className="hover:text-gray-300">Contact</a>
+          <a href="#" className="hover:text-gray-300">
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)